ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4611,4 +4611,241 @@ AHU_DFSS_DX2ZC_ECONZ_CO2C_DSP_RCKTM2X_HTZC_CC2XM_RTM_DTM:
   - RTM
   - DTM
   opt_uses:
-  - condensing_fan_current_sensor
\ No newline at end of file
+  - condensing_fan_current_sensor
+  
+
+
+#PAC Unit#  
+AHU_CHWPM_SFSS_DXSC_SFRDM_CPRDM_EVFSC_STC_ZLTM_ZLPM_ZLHM_DXSC_SFRMM_CHWDT_ZTM:
+  description: "AHUs with chilled water temperature monitoring and compressor monitoring with zone temperature and humidity monitoring ."
+  is_canonical: true
+  implements:
```

Comment:
Where is the outside air section? AHUs must control outside and return air, so without it this is incomplete. Please add the data for outside air control.